projects/kalidea/kaligraphi/src/lib/molecules/kal-list/kal-selection.ts

```diff
@@ -0,0 +1,129 @@
+import { SelectionModel } from '@angular/cdk/collections';
+import { cloneDeep, isNil } from 'lodash';
+
+export class KalSelection<T extends { id?: string }> {
+
+  count ? = 0; // Total of elements
```

Comment:
Is the editor able to define theses properties with its comment ?
Maybe you should set comments with /** */